_posts/2019-12-16-database-migration-in-self-contained-systems.md

```diff
@@ -0,0 +1,265 @@
+---
+
+title: Database Migration in Self Contained Systems
+categories: [craft]
+modified: 2019-12-16
+author: artur
+excerpt: "A tutorial on how to migrate databases in SCS"
+image:
+  auto: 0066-database-migration
+---
+
+When you develop a application nowadays, you as programmer just want to get the database and 
+to add new scripts without thinking about database maintenance. This problem is already solved
+by frameworks like [flyway](https://flywaydb.org/) or [liquibase](https://www.liquibase.org/). 
+The database is provided to you in this case or you have to take care about this.
```

Comment:
I don't understand the value of this sentence. Please either elaborate or delete it.

Reply:
rewritten